### ISSUE TITLE
Delete master branch

### ISSUE THREAD

**Issue by GrahamCampbell** [cartalyst/stripe#181]
It isn't needed. We already have a 2.4 branch, which serves to replace the branch named master. Using proper branch names like `2.4` avoids the need to use branch aliases in the composer.json file, and makes it super clear to contributors what version the branch is.

**Comment by GrahamCampbell** [comment]
Also, should the 3.x branch be renamed to `3.0`?

**Comment by brunogaspar** [comment]
No, it's supposed to be 3.x, and why should i delete `master`? Not really following your idea here.

**Comment by GrahamCampbell** [comment]
Because `3.x` and `master` are both bad branch names, and don't correspond to the version properly: thus requiring branch aliases to tell composer the major and minor version. Given that there is already a 2.4 branch and a 3.x branch, master serves no purpose. It just exists and is out of date. Moreover, the word master is no longer considered politically correct for use in software.

TLDR: Using major.minor for branch names is clear, inclusive, and avoids the need to use (and maintain) branch aliases.
